.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+on:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
```

Comment:
Where is this matrix?

Reply:
I'm returning you the question hahaha

Reply:
https://github.com/DataDog/datadog-api-client-python/blob/cdd884303b2c2d7959cff6890a7a36d1173bd0b6/.github/workflows/release.yml#L11